templates/web-deployment.yaml

```diff
@@ -48,21 +48,21 @@ spec:
       {{- end }}
       {{- end }}
       initContainers:
-      - name: concourse-migration
-        {{- if .Values.imageDigest }}
-        image: "{{ .Values.image }}@{{ .Values.imageDigest }}"
-        {{- else }}
-        image: "{{ .Values.image }}:{{ .Values.imageTag }}"
-        {{- end }}
-        args: ["migrate", "--migrate-to-latest-version"]
-        env:
+        - name: concourse-migration
+          {{- if .Values.imageDigest }}
+          image: "{{ .Values.image }}@{{ .Values.imageDigest }}"
+          {{- else }}
+          image: "{{ .Values.image }}:{{ .Values.imageTag }}"
+          {{- end }}
+          args: ["migrate", "--migrate-to-latest-version"]
+          env:
 {{- include "concourse.postgresql.env" . | indent 10 }}
-        volumeMounts:
-        {{- if not (eq .Values.concourse.web.postgres.sslmode "disable") }}
-          - name: postgresql-keys
-            mountPath: {{ .Values.web.postgresqlSecretsPath | quote }}
-            readOnly: true
-        {{- end }}
+          volumeMounts:
```

Comment:
I actually would leave everything else intact and just move the `volumeMounts:` line below the `if` statement on line 61. Otherwise it's just a dangling `volumeMounts:` node with nothing within, which isn't very useful anyway. Or set `volumeMounts: []` `{{- if (eq .Values.concourse.web.postgres.sslmode "disable") }}`